uint8   id
uint8	anchor_id
uint8   sequence
uint64  timePollSent
uint64  timePollAckReceived
uint64  timeRangeSent